### PR TITLE
create common materials models

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
@bqpd not sure of the best way to go about doing this, see use case in convexengineering/solar@47040c1757d303c0060481b8f6ac90fc406d202c

**Comment by mjburton11** [comment]
@bqpd One problem I'm having that you'll see the in the solar commit, is that I want the wing skin material, htail skin and vtail skin materials models to all be the same.  I can do that with this approach, but the materials model that is used is like the 5th one that is created.  I think because importing a model causes the materials model to be created? I'm wondering if there's a better way to do this.

**Comment by bqpd** [comment]
(noting here that we met and discussed in lab doing this with instances created on import of gplibrary)

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/wing/wing_test.py [hunk 1]
```diff
@@ -77,45 +75,10 @@ def box_spar():
         W, fs, perf, loading])
     m.solve(verbosity=0)
 
-def materials():
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/wing_test.py]: why'd you remove the test?

File: gpkitmodels/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from gpkit import Variable
+g = Variable("g", 9.81, "m/s^2", "gravitational acceleration")
```
Comment (bqpd) [inline gpkitmodels/__init__.py]: add "near the earth's surface" ?

File: gpkitmodels/GP/aircraft/wing/wing_test.py (hunk 1 above)
Comment (mjburton11) [reply gpkitmodels/GP/aircraft/wing/wing_test.py]: This tested that I could swap out the materials models to have a common material model for any use of CFRPFabric, but this became irrelevant once you import created models from the materials `__init__.py`

File: gpkitmodels/__init__.py (hunk 2 above)
Comment (mjburton11) [reply gpkitmodels/__init__.py]: hmm or just "earth surface?"

Comment (bqpd) [reply gpkitmodels/__init__.py]: sounds good to me!

File: gpkitmodels/GP/aircraft/wing/wing_test.py (hunk 1 above)
Comment (bqpd) [reply gpkitmodels/GP/aircraft/wing/wing_test.py]: ah, got it

File: gpkitmodels/__init__.py (hunk 2 above)
Comment (mjburton11) [reply gpkitmodels/__init__.py]: I pushed directly to master.  Sorry I should have asked you to review this again before merging...

**Comment by bqpd** [comment]
naw worries!

**Pull request merged**